SIMPLE_WEBHOOK_SETUP.md

```
@@ -0,0 +1,61 @@
+- **"Events"**: Checkboxes → Check only `INSERT`
+
+### 4. **HTTP Headers Section**
+**What you see:**
+```
+Content-type: application/json ✅ (already correct)
+```
+
+### 5. **Request Body Section** (This is the important part!)
+**What you'll see:** A large text box labeled "Request Body" or "Payload"
+
+**Copy and paste this exactly:**
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)